app/common/src/text/english.json

```diff
@@ -365,10 +365,13 @@
   "startWithATemplate": "Discover",
   "openInfoMenu": "Open info menu",
   "noProjectIsCurrentlyOpen": "No project is currently open.",
-  "versionOutdatedTitle": "Upgrade Enso Now",
-  "versionOutdatedPrompt": "Download the latest version to get the latest upgrades and Cloud functionality.",
+  "versionOutdatedTitle": "Update available",
+  "versionOutdatedPrompt": "The new version of Enso is available. Upgrade to get the latest upgrades and Cloud functionality.",
   "yourVersion": "Your version:",
-  "latestVersion": "Latest version:",
+  "latestVersion": "$0 ($1)",
+  "changeLog": "View changelog",
+  "downloadingAppMessage": "We are started downloading the latest version of Enso. This may take a while depending on your internet speed. Please check your downloads folder. You may close this dialog.",
```

Comment:
`We are started` -> `We have started`

nit: i think active voice is typically considered less professional - so `The latest version of Enso is now downloading` rather than using "we" ay be preferred.

nit 2: also the sentences are quite short which makes them (IMO) quite... disjointed? like they stop and start a lot.

nit 3: we may want to open the downloaded file when it is ready - but maybe that should wait until we have a way to automatically run the installer and close this instance (or have the installer require the instance to be closed before continuing)